front/src/reducers/authorization/listener.ts

```diff
@@ -0,0 +1,15 @@
+import { isRejectedWithValue } from '@reduxjs/toolkit';
+
+import { type AppStartListening } from 'store/listenerMiddleware';
+
+export default function add403HttpErrorListener(startListening: AppStartListening) {
+  startListening({
+    matcher: isRejectedWithValue,
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    effect: (action: any) => {
```

Comment:
I think we could avoid this `any` by using `UnknownAction` type, I'll notice you if I find anything relevant